### PR TITLE
add missing `return` statement to sample code

### PR DESCRIPTION
**Pull request by dd-ssc**
The `return` statement is present in the "long" / "final" version of the sample code (albeit with a weird indentation), but is missing in the snippets that precede it; without it, the code fails with

    Failed to compile.

    ./src/App.js
      Line 30:  Expected an assignment or function call and instead saw an expression  no-unused-expressions

    Search for the keywords to learn more about each error.

which leaves any non-JavaScript experts (such as myself) with a big question mark over their heads...

### CODE REVIEW

File: js/api.md [hunk 1]
```diff
@@ -391,8 +391,9 @@ class App extends React.Component {
             <Connect query={graphqlOperation(queries.listTodos)}>
                 {({ data: { listTodos }, loading, error }) => {
                     if (error) return (<h3>Error</h3>);
-                    if (loading || !listTodos) return (<h3>Loading...</h3>);
-                    return (<ListView todos={listTodos.items} /> );
+                    if (loading || !listTodos) return (<h3>Loading...</h3>)
```
Comment (elorzafe) [inline js/api.md]:
```suggestion
                    if (loading || !listTodos) return (<h3>Loading...</h3>);
```

File: js/api.md [hunk 2]
```diff
@@ -391,8 +391,9 @@ class App extends React.Component {
...
+                    
+                    return <ListView todos={listTodos.items} />
```
Comment (elorzafe) [inline js/api.md]:
```suggestion
                    return <ListView todos={listTodos.items} />;
```

File: js/api.md [hunk 3]
```diff
@@ -418,7 +419,8 @@ Also, you can use the `subscription` and `onSubscriptionMsg` attributes to enabl
     {({ data: { listTodos }, loading, error }) => {
         if (error) return (<h3>Error</h3>);
         if (loading || !listTodos) return (<h3>Loading...</h3>);
-        return (<ListView todos={listTodos ? listTodos.items : []} />);
+
+        return <ListView todos={listTodos ? listTodos.items : []} />
```
Comment (elorzafe) [inline js/api.md]:
```suggestion
        return <ListView todos={listTodos ? listTodos.items : []} />;
```

File: js/api.md [hunk 4]
```diff
@@ -505,7 +507,8 @@ class App extends Component {
         {({ data: { listTodos }, loading, error }) => {
           if (error) return <h3>Error</h3>;
           if (loading || !listTodos) return <h3>Loading...</h3>;
-            return (<ListView todos={listTodos.items} />);
+
+          return <ListView todos={listTodos.items} />
```
Comment (elorzafe) [inline js/api.md]:
```suggestion
          return <ListView todos={listTodos.items} />;
```

**Comment by jordanranz** [comment]
I don't believe the docs example code is the problem. Gonna go ahead and close this as the fork repo is unknown.

Thanks for contributing.

**Pull request closed without merging**